OSDOCS#12811: [OLMv1]  Replace port-forwarding cmd w/ `opm render` (#85686)
openshift/openshift-docs: snippets/olmv1-on-cluster-catalog-queries.adoc

// Text snippet included in the following modules:
//
// * olmv1-finding-operators-to-install.adoc

:_mod-docs-content-type: SNIPPET

> Currently, {olmv1-first} does not support querying on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.

Author: I am not sure if I got this right. And if I got it right, I am not sure if this is the best phrasing. Please let me know your thoughts, Operator Framework team.

> Currently, {olmv1-first} does not support querying on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.

Reviewer: ```suggestion
Currently in {olmv1-first}, you cannot query on-cluster catalogs managed by catalogd. In OLM v1, you must use the `opm` and `jq` CLI tools to query the catalog registry.
```
Just a tweak to make both sentences in this snippet address the reader ("you") instead of focusing on the product.
